src/controllers/userController.js

```diff
@@ -0,0 +1,130 @@
+import moment from 'moment';
+import crypto from 'crypto';
+import Validation from '../validation';
+import { sendResetMail, sendSignupMail } from '../services/sendMail';
+import { errorResponse, successResponse } from '../utils/response';
+import Hash from '../utils/hash';
+import models from '../models';
+
+const { User, Login, Reset } = models;
+
+export default class UserController {
+  /**
+   * @description Generate link to reset a user password
+   * @static
+   * @param {*} req
+   * @param {*} res
+   * @returns Promise {UserController} A reset link for new password
+   * @memberof UserController
+   */
+  static async forgotPassword(req, res) {
+    try {
+      const { errors, isValid } = Validation.validateEmail(req.body);
+
+      // Check validation
+      if (!isValid) {
+        return errorResponse(res, 400, errors);
+      }
+
+      const { email } = req.body;
+
+      // Find user by email
+      req.user = await User.findOne({ where: { email } });
+      const { user } = req;
+      // Check for user
+      if (!user) {
+        sendSignupMail(email);
```

Comment:
The `click link to signup` is not working.

Reply:
Yes it doesn't work yet until the signup route is handled. I believe someone else has been assigned the task.

Reply:
ok